### DISCUSSION TITLE
[Community standards] Picking a code of conduct for COAsT (and COAsT-site)

### DISCUSSION THREAD

**Discussion by thogar-computer** [British-Oceanographic-Data-Centre/COAsT#533] [General]
# Intro
There is a checklist of items that GitHub has picked as 'good practice' for Open Source projects to follow; 

https://github.com/British-Oceanographic-Data-Centre/COAsT/community

we have some of these already and it would be good to get a full house 

## User Story 
As a contributor, I want to know what expected behavior myself and others should follow when working within this community so that we can have a constructive and robust discussion to improve the code for everyone.

Given a the COAst (and COAsT-site) repo are public and open to all
When a new contributor joins
Then they know what is expected in terms of behavior

## details 
Here we are looking to pick and display a code of conduct for the COAsT and COAst-site repos. There are a lot of choices and we could even write our own. 

it is, however, more in fitting with the OS community to adopt an existing code if it is fit for our purpose. 

there are two which have good traction; 

1) https://www.contributor-covenant.org/version/2/1/code_of_conduct/
2) https://web.archive.org/web/20200330154000/http://citizencodeofconduct.org/

### AC
This ticket is done, when both the COAsT and COAsT-site repos have a code of conduct setup

**Comment by thogar-computer** [comment]
@lukegorman 
@roje-bodc 
@jpolton 

so there is a code of conduct in the pull request - I have gone for the covenant for now 

@jpolton 
rather than closing this ticket, we can move this to a discussion for the community (if you wish) and merge #531 as a starter for ten

**Comment by jpolton** [comment]
@thogar-computer. Hmm. I approve closing the ticket pending adding a contact address in line 60-something.

The community can still discuss it even if we have a policy in place.
